src/core/dbus-manager.c

```diff
@@ -2914,6 +2914,7 @@ static int aux_scope_from_message(Manager *m, sd_bus_message *message, Unit **re
                 return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS,
                                          "Name \"%s\" of auxiliary scope doesn't have .scope suffix.", name);
 
+        log_unit_warning(from, "Deprecated StartAuxiliaryScope() call");
```

Comment:
Can you make this a proper english sentence, i.e. "D-Bus call StartAuxiliaryScope() has been invoked which is deprecated. Removal is imminent."